server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java

```diff
@@ -899,4 +906,50 @@ public void removeScanServerRanges(ServerContext context) {
     }
   }
 
+  void moveTableProperties(ServerContext context) {
```

Comment:
Am I correct in understanding that this moves table properties from the system config to all currently existing namespaces?

Reply:
Yes, it moves the table properties in the system configuration to all known namespaces, unless the property already exists there with a different value.